### PR TITLE
Open beta document changes

### PR DESCRIPTION
**Pull request by jakon2048**
Updated documentation and fixed some bigs for Open Beta.

### CODE REVIEW

File: requirements.md [hunk 1]
```diff
@@ -6,55 +6,44 @@ permalink: /docs/requirements
...
-* El Capitan (10.11) or later
+* El Capitan (10.11) or later.
 
 ### Linux
 
-* Ubuntu 'Trusty Tahr' (14.04) or later recommended
-* Java 64-bit JDK or JRE 11 (or newer)
-* [Vulkan] GPU drivers for desktop frame profiling
+* Ubuntu 'Trusty Tahr' (14.04) or later recommended.
+* Java 64-bit JDK or JRE 11 (or newer).
```
Comment (hevrard) [inline requirements.md:45]: 11 -> 8 (we had some shenanigans, we're back to Java 8 now)

File: requirements.md [hunk 2]
```diff
@@ -6,55 +6,44 @@ permalink: /docs/requirements
...
 
 The following are the minimum system requirements for profiling an Android
 application using a desktop or laptop:
 
 ### Windows
 
-* Windows 7 or later
-* [Vulkan] GPU drivers for desktop frame profiling
+* Windows 7 or later.
+* [Vulkan] GPU drivers for desktop frame profiling.
```
Comment (hevrard) [inline requirements.md:36]: For the "we're a profiler" message, do not mention frame profiling, or even profiling at all, on desktop. So remove this line.

File: requirements.md [hunk 3]
```diff
@@ -6,55 +6,44 @@ permalink: /docs/requirements
...
+* El Capitan (10.11) or later.
 
 ### Linux
 
-* Ubuntu 'Trusty Tahr' (14.04) or later recommended
-* Java 64-bit JDK or JRE 11 (or newer)
-* [Vulkan] GPU drivers for desktop frame profiling
+* Ubuntu 'Trusty Tahr' (14.04) or later recommended.
+* Java 64-bit JDK or JRE 11 (or newer).
+* [Vulkan] GPU drivers for desktop frame profiling.
```
Comment (hevrard) [inline requirements.md:46]: Remove this line: we don't have proper profiling support on desktop.

File: validation.md [hunk 4]
```diff
@@ -5,21 +5,36 @@ title: Device Support Validation
...
+Do not disturb the device while device support validation is in progress as it may cause the device to fail the
 validation checks. If a device fails validation, but is setup correctly, you can retry the
 support validation by re-selecting the device.
 </div>
+
+## Why is my device not supported?
+
+AGI needs a compatible GPU driver to run. To ensure the tool provides good and valid profiling data, we run a validation check when you first connect a new device.
+
+If your device is listed in the supported device page, you can expect AGI to pass validation.
```
Comment (hevrard) [inline validation.md:18]: Add link to supported devices page.

File: validation.md [hunk 5]
```diff
@@ -5,21 +5,36 @@ title: Device Support Validation
...
+
+## When will AGI support my device?
+
+We will also periodically update our supported device page with devices that support AGI. We are working with our OEM partners to add support to more devices.
+
+Please [file an issue] in our GitHub repository to request support for a device. We will connect you to the OEM in question.
+
+## What should I expect? I use a supported device.
+
+Device validation takes about 10 seconds, during which you should see a spinning cube on the device screen. Once a device passes the validation process, it can be used to trace and profile Android applications.
```
Comment (hevrard) [inline validation.md:30]: ... used to trace and profile ...  -> used to profile

File: getting-started.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+8. Under "Start and Duration" section, set "Start at" to "Manual", and "Duration" to 2.
+
+9. In the "Trace Option" section, click on the "Configure" button to pop-up a configuration window where you can select which performance counters you want to be captured.
+
+    <img src="../images/system_profiler/capture-config.png" width="500px">
+
+10. In the GPU section, click on "Select" to bring up the counter selection dialog. Click on "default" to select the set of default counters and press "OK".
+
+    <img src="../images/system_profiler/counter-config.png" width="500px">
```
[the remark this answers is not in the document]
Comment (jakon2048) [reply getting-started.md:76]: we should update it later once this is irrelevant

**Review by pmuetschard — APPROVED** [review]

**Pull request merged**